src/integrationTest/java/uk/gov/hmcts/reform/sscs/service/EvidenceShareServiceIt.java

```diff
@@ -184,28 +197,26 @@ public void appealWithMrnDateOlderThan30Days_shouldGenerateDL16TemplateAndAndAdd
         assertEquals("filename1.pdf", documentCaptor.getValue().get(2).getName());
 
         verify(restTemplate).postForEntity(anyString(), any(), eq(byte[].class));
-        verify(evidenceManagementService).upload(any(),  eq("sscs"));
+        verify(evidenceManagementService).upload(any(), eq("sscs"));
         verify(ccdService).updateCase(any(), any(), any(), any(), eq("Uploaded dl16-12345656789.pdf into SSCS"), any());
         verify(bulkPrintService).sendToBulkPrint(any(), any());
 
         verify(ccdService).updateCase(any(), any(), eq(EventType.SENT_TO_DWP.getCcdType()), any(), eq(documentList), any());
     }
 
-    @Test(expected = NoMrnDetailsException.class)
+    @Test
     public void appealWithNoMrnDate_shouldNotGenerateTemplateOrAddToCcd() throws IOException {
         assertNotNull("evidenceShareService must be autowired", evidenceShareService);
         String path = Objects.requireNonNull(Thread.currentThread().getContextClassLoader()
             .getResource("appealReceivedCallback.json")).getFile();
         String json = FileUtils.readFileToString(new File(path), StandardCharsets.UTF_8.name());
 
-        try {
-            evidenceShareService.processMessage(json);
-        } catch (NoMrnDetailsException e) {
-            verifyNoMoreInteractions(restTemplate);
-            verifyNoMoreInteractions(evidenceManagementService);
-            verifyNoMoreInteractions(ccdService);
-            throw e;
-        }
+        evidenceShareService.processMessage(json);
+        verifyNoMoreInteractions(restTemplate);
+        verifyNoMoreInteractions(evidenceManagementService);
+        then(ccdService)
+            .should(times(1))
+            .updateCase(any(), any(), eq("sendToDwp"), any(), any(), any());
```

Comment:
Can we capture the new `BulkPrintInfo` with expected output here?

Reply:
bulkPrintInfo is not an argument of that ccdService mock so we cannot capture. I have captured the caseData to ensure we are on the expected scenario and that therefore the case does not have mrn date and that we flag the error in the case and that the processMessage result is empty.

Reply:
👍